cernopendata/modules/fixtures/data/docs/delphi-news-first-release/delphi-news-first-release.md

```diff
@@ -0,0 +1,8 @@
+The DELPHI Collaboration decided to open their data and release it for the Open Data Portal.
```

Comment:
I'd rather say "release it to the physics community for public access".  The portal is only the "means" how to do the publishing.

Anyway, the content of the release announcement will be probably known much later. For now I would probably just remove it from the pull request, or at least put a `FIXME` placeholder so that any casual visitors would not have any wrong impression about this page!